mongo/datadog_checks/mongo/mongo.py

```diff
@@ -7,7 +7,7 @@
 from distutils.version import LooseVersion
 
 import pymongo
-from six import PY3, itervalues
+from six import itervalues
```

Comment:
I think you can also get rid of `itervalues` and use `values()` directly since it looks like we do not support python 2 here

Reply:
https://github.com/DataDog/integrations-core/pull/12767/commits/5c40b4cd7798cb0647ffb5dbc2aa7572dfd0eb0f